protocol/contracts/beanstalk/diamond/DiamondCutFacet.sol

```diff
@@ -23,7 +24,7 @@ contract DiamondCutFacet is IDiamondCut {
         FacetCut[] calldata _diamondCut,
         address _init,
         bytes calldata _calldata
-    ) external override {
+    ) external override fundsSafu {
```

Comment:
It is a very good point. My reaction to the first paragraph was 'we can just diamond cut the restriction away' but if we can so could an attacker. it will not protect anything but could be an inconvenience.

Reply:
this was removed but i lost which commit exactly